lib/Mojolicious/Plugin/Webpack.pm

```diff
@@ -18,7 +18,7 @@ sub route      { shift->{route} }
 
 sub register {
   my ($self, $app, $config) = @_;
-  $self->{route} = $app->routes->route('/asset/*name')->via(qw(HEAD GET))->name('webpack.asset');
+  $self->{route} = $app->routes->any('/asset/*name')->methods(qw(HEAD GET))->name('webpack.asset');
```

Comment:
This will break old versions of Mojolicious. Please have a look at https://github.com/jhthorsen/mojolicious-plugin-assetpack/pull/142/files to see how it can be done without breaking compatibility.

Reply:
I see you fix  this on master branch, thanks